examples/Braccio_Learn_and_Repeat/Braccio_Learn_and_Repeat.ino

```diff
@@ -25,10 +28,12 @@ static const char * btnm_map[] = { "RECORD", "\n", "REPLAY", "\n", "ZERO_POSITIO
 
 
 static void eventHandlerMenu(lv_event_t * e) {
+  Braccio.lvgl_lock();
   lv_event_code_t code = lv_event_get_code(e);
   lv_obj_t * obj = lv_event_get_target(e);
 
   if (code == LV_EVENT_KEY && lv_indev_get_key(lv_indev_get_act()) == LV_KEY_HOME) {
```

Comment:
This if statement is crashing the code when I do as in this [PR](https://github.com/arduino-libraries/Arduino_Braccio_plusplus/blob/d8b4421c73bce5ea7fe3bc451592cd940928b4d8/examples/Braccio_Learn_and_Repeat/Braccio_Learn_and_Repeat.ino#L35), everything works fine, and the code didn't crash.